sdk/compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -86,6 +86,8 @@ data Config = Config
         -- ^ Information about dependencies (not data-dependencies)
     , configSdkPrefix :: [T.Text]
         -- ^ prefix to use for current SDK in data-dependencies
+    , configIgnoreExplicitExports :: Bool
+        -- ^ Should explicit export information be disregarded, and all definitions be exported
```

Comment:
(for data deps)

Reply:
At this point we're in the context of a single data dep, so this is really for "this package"